src/digisos/hovedmeny/paneler/PabegynteSoknader.tsx

```diff
@@ -21,14 +21,14 @@ const PabegyntSoknad = ({
     deleteDate: Date;
     antallPabegynteSoknader: number;
 }) => {
-    const history = useHistory();
+    const navigate = useNavigate();
 
     const onPabegyntSoknadClick = (event: React.SyntheticEvent, href: string) => {
         event.preventDefault();
         logAmplitudeEvent("Klikk på påbegynt søknad", {
             antallPabegynteSoknader,
         });
-        history.push(href);
+        navigate(href);
```

Comment:
ser ut som denne blir feil siden man står på /informasjon, slik at ny url da blir /informasjon/skjema/{id}/1 ?

Reply:
Right you are! Fikset i ee75aae. :)

Den langsiktige løsningen på dette er å oppgradere Amplitude-SDK og tilpasse logAmplitudeEvent så det ikke returnerer for tidlig, jamf https://trello.com/c/G67lubO5/48-migrere-fra-amplitude-js-til-amplitude-analytics-browser